cmd/snap/cmd_run.go

```diff
@@ -485,6 +487,20 @@ func straceCmd() ([]string, error) {
 	}, nil
 }
 
+func (x *cmdRun) runCmdUnderGdb(origCmd, env []string) error {
+	env = append(env, "SNAP_CONFINE_RUN_UNDER_GDB=1")
```

Comment:
Can you please document this variable somewhere (e.g. on the forum) as well as add a small note here.